### PR TITLE
feat(vote credit): add card, modal and row components

### PR DESCRIPTION
**Pull request by VERZUOL1**

### CODE REVIEW

**Review by stee1house — APPROVED** [review]

File: astro_2.0/features/pages/myAccount/cards/VoteCreditCard/components/VoteCreditRow/VoteCreditRow.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  const isMobile = useMedia('(max-width: 640px)');
+
+  const [showModal] = useModal(VoteCreditModal);
+
+  const handleAssign = useCallback(async () => {
+    await showModal({
+      daoName: dao.name || dao.id,
+      daoFunds: dao.totalDaoFunds,
+    });
+  }, [dao.id, dao.name, dao.totalDaoFunds, showModal]);
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/pages/myAccount/cards/VoteCreditCard/components/VoteCreditRow/VoteCreditRow.tsx:29]: Is there a reason not to destruct it and use `id`, `name`, etc... We are duplicating dao all the time.

File: astro_2.0/features/pages/myAccount/cards/VoteCreditCard/components/VoteCreditModal/VoteCreditModal.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              </div>
+            </InputWrapper>
+          </div>
+
+          <div className={styles.footer}>
+            <Button
+              capitalize
+              className={styles.confirmButton}
+              variant="secondary"
+              onClick={() => onClose()}
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/pages/myAccount/cards/VoteCreditCard/components/VoteCreditModal/VoteCreditModal.tsx:84]: Can we just pass onClose?

File: astro_2.0/features/pages/myAccount/cards/VoteCreditCard/components/VoteCreditModal/VoteCreditModal.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+            >
+              Cancel
+            </Button>
+            <Button capitalize type="submit" className={styles.confirmButton}>
+              Submit
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/pages/myAccount/cards/VoteCreditCard/components/VoteCreditModal/VoteCreditModal.tsx:89]: Should we leave some comment to remember about localisation? Maybe we should introduce some TODO at the top of the unlocalised files?

File: astro_2.0/features/pages/myAccount/cards/VoteCreditCard/VoteCreditCard.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    if (isMounted()) {
+      setAccountDaos(daosList);
+    }
+  }, [accountId, isMounted]);
+
+  useEffect(() => {
+    (async () => {
+      await getDaosList();
+    })();
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/pages/myAccount/cards/VoteCreditCard/VoteCreditCard.tsx:32]: Is there sense to wait till we get daos? Will we face some "race condition" if we remove await?

File: astro_2.0/features/DaoFeatureInfo/DaoFeatureInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+              ...settings.features,
+              [featureKey]: true,
+            }
+          : {
+              [featureKey]: true,
+            },
+      };
+
+      const publicKey = await nearService?.getPublicKey();
+      const signature = await nearService?.getSignature();
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/DaoFeatureInfo/DaoFeatureInfo.tsx:69]: We have a `getPublicKeyAndSignature` method to get both)

**Pull request merged**